cmd/aro/operator.go

```diff
@@ -124,5 +125,12 @@ func operator(ctx context.Context, log *logrus.Entry) error {
 	// +kubebuilder:scaffold:builder
 
 	log.Info("starting manager")
+
+	s := liveness.New("8080") //creates a route on :8080 for healthz/ready for liveness probe
+
+	go func() {
+		log.Warn(s.ListenAndServe())
+	}()
```

Comment:
just do a direct copy/paste of the existing code please:

```
	l, err := net.Listen("tcp", ":8080")
	if err != nil {
		return err
	}

	go func() {
		_ = http.Serve(l, http.HandlerFunc(func(http.ResponseWriter, *http.Request) {}))
	}()
```

rationale: ensures we completely fail up front if we can't listen on 8080 for any reason.

`func(http.ResponseWriter, *http.Request) {})` will return 200 OK on any request, including /healthz/ready.